### DISCUSSION TITLE
Home Assistant: TS011F  not supported by Dev Z2M when it should be?

### DISCUSSION THREAD

**Discussion by CheekyMonkeyPo** [Koenkk/zigbee2mqtt#13941] [Devices]
Apologies if this is incorrect or a simple matter. I'm trying to use two plugs with power monitoring that pair as TS011F with Z2M, but they come up as not suported. According to Blackadder and these forums, they are supported by both ZHA and Z2M. I've tried with ZHA and they seem to work well, but that route wipes out my Z2M setup as it pulls in all the other Zigbee devices and my setup fails. But at least the plugs seem to work.

 I'm using the edge version of Z2M and everything else works great. I'm just confused by the lack of support. They are by Ajax ( TZ3000_1h2x4akh) and I've tried to create a converter following the instuctions but I'm stuck as the debug output doesn't list the information needed. I'm on Z2M dev version 1.21.2-dev commit: 8ac3989. I can see a more recent version but to my embarrassment I can't see how to update it either!

I'm guessing this is a rookie issue but I'm a bit stuck now. Any idea if it's possible to use these plugs with Z2M in home assistant? 

Thanks!

Paul.

**Comment by CheekyMonkeyPo** [comment]
Worked out how to update- was confused by the auto-update option. Now plugs are recognised. Needed to pull the power on my pi before Z2MQTT wiould install and open, but then works ok. Reboot didn't do enough oddly.

**Reply by xingpao** [reply]
Hi, same issue here. I did not really understood how you’ve solved it …. Could you please explain it with more details? Thanks!

**Comment by CheekyMonkeyPo** [comment]
No problem. I first needed to update my Edge version of Z2M (the developers version).   From memory this was through uninstalling and reinstalling Z2M Edge as per the manual, as there's no automatic update function.  Remember to make the .config backup and reload as described in the manual or you'll lose all your settings. Then I  restarted the system after pulling the plug. A simple restart or reboot didn't work  but pulling the cord then starting again for. After that, it's worked brilliantly. 

Hope this helps 😃

**Reply by xingpao** [reply]
Thanks, this is now ok! It worked for me too.

**Comment by CheekyMonkeyPo** [comment]
Hurrah! Great to hear that. I assumed it auto- updated but once I understood it did not, it made more sense then.

Happy Home Assistanting!
